router/pump.go

```diff
@@ -153,67 +153,25 @@ func (p *LogsPump) pumpLogs(event *docker.APIEvents, backlog bool) {
 		debug("pump.pumpLogs():", id, "ignored: environ ignore")
 		return
 	}
-	if !logDriverSupported(container) {
-		debug("pump.pumpLogs():", id, "ignored: log driver not supported")
-		return
-	}
 
-	var sinceTime time.Time
-	if backlog {
-		sinceTime = time.Unix(0, 0)
-	} else {
-		sinceTime = time.Now()
-	}
-
-	p.mu.Lock()
-	if _, exists := p.pumps[id]; exists {
-		p.mu.Unlock()
-		debug("pump.pumpLogs():", id, "pump exists")
-		return
-	}
 	outrd, outwr := io.Pipe()
 	errrd, errwr := io.Pipe()
 	p.pumps[id] = newContainerPump(container, outrd, errrd)
 	p.mu.Unlock()
 	p.update(event)
 	go func() {
-		for {
```

Comment:
This kills the work done to fix the fact that we previously didn't reconnect, as mentioned here: https://github.com/gliderlabs/logspout/pull/191